src/fsharp/vs/ServiceUntypedParse.fs

```diff
@@ -990,19 +990,24 @@ module UntypedParseImpl =
             
             let rec getKind isClass isInterface isStruct = 
                 function
-                | [] -> isClass, isInterface, isStruct
-                | (SynAttr "Class")::xs -> getKind true isInterface isStruct xs
-                | (SynAttr "AbstractClass")::xs -> getKind true isInterface isStruct xs
-                | (SynAttr "Interface")::xs -> getKind isClass true isStruct xs
-                | (SynAttr "Struct")::xs -> getKind isClass isInterface true xs
-                | _::xs -> getKind isClass isInterface isInterface xs
```

Comment:
I guess this is a typo? Two time `isInterface`

Reply:
oh, looks like this PR has just become not so useless. wow